ipmininet/cli.py

```diff
@@ -81,6 +81,25 @@ def do_ping6pair(self, _line):
         """Ping (IPv6-only) between first two hosts, useful for testing."""
         self.mn.ping6Pair()
 
+    def do_linkFailure(self,line):
```

Comment:
You should document more precisely this command, given you are not only failing the link per say (i.e., simulating a faulty amplifier/switch on the path) but really downing an interface.

This causes internal events to be generated, and applications can listen to them (e.g., with netlink RTM_DELLINK https://linux.die.net/man/7/rtnetlink) causing instant failure recovery for them--i.e., experimenters need to add a L2 switch in between and fail it instead.